lib/publiccloud/azure.pm

```diff
@@ -633,13 +632,20 @@ This method is called called after each test on failure or success to revoke the
 
 sub cleanup {
     my ($self, $args) = @_;
+    script_run('cd ~/terraform');
+    my $terraform_output = script_output('terraform output -json', proceed_on_failure => 1);
+    my $terraform_output_json = decode_json($terraform_output);
+    my $instance_id = $terraform_output_json->{instance_id}->{value}[0];
+    $instance_id =~ s/.*\/(.*)/$1/;
+    my $resource_group = $terraform_output_json->{resource_group_name}->{value}[0];
+    script_run('cd');
+
```

Comment:
Are you sure we want to run these commands before `select_host_console` below?

Reply:
Good question. It is the case in EC2 but I'll move it after we're for sure in host console.